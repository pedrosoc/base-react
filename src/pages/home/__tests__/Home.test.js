import React from "react";
import { shallow } from "enzyme";

import { Home } from "../";

it("snapshot test for home component", () => {
	const home = shallow(<Home />);
	expect(home.getElement()).toMatchSnapshot();
});
