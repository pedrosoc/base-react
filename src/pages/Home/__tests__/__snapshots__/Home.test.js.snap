// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for home component 1`] = `
<ForwardRef(Styled(Container))>
  <h4>
     
    In√≠cio
     
  </h4>
</ForwardRef(Styled(Container))>
`;
